src/client/Client.cc

```diff
@@ -431,6 +431,8 @@ int Client::init()
 
   client_lock.Unlock();
 
+  g_conf->add_observer(this);
```

Comment:
Heh, this should be cct, not g_conf — build failure! Guess my advice was a little slapdash. ;)